country/ireland.md

```diff
@@ -1,27 +1,21 @@
 # Ireland
-
-Ireland has 2 laws that govern email marketing and other forms of electronic marketing. These are the Irish Data Protection Act 2018 and EU GDPR. Both laws combined cover all aspects of electronic marketing in Denmark.
+Ireland has two laws that govern email marketing and other forms of electronic marketing; the Irish Data Protection Act 2018 and EU GDPR.
 
 ## Content required
-
 All emails sent for marketing purposes should have a clear identification of the sender name, mailing address and a clear identification of the sender. The law requires you to have these as compulsory data to be mentioned in your marketing emails.
 
 ## Consent
-
 You can only send marketing emails to those who have provided explicit consent to receive marketing emails from you.
 
 This can be either by having a double opt-in or by checking an explicit checkbox while subscribing with your privacy policy link mentioned there.
 
 ## Notable exceptions or requirements
-
 As Ireland follows GDPR, it requires that companies have a data security officer. The person is in charge of maintaining and enforcing data security standards.
 
 ## Penalties
+The maximum fine available under the GDPR is up to 20 million EUR, or 4% annual global turnover – whichever is higher. The "Data Protection Commissioner" is the supervisory authority.
 
-The maximum fine available under the GDPR is up to €20 million, or 4% annual global turnover – whichever is higher. The "Data Protection Commissioner" will be  the supervisory authority here.
-
-In addition to a fine under GDPR, the  Irish government will also impose a fine up to EUR 250,000 per message sent by a company and an individual may be fined up to EUR 50,000 per message
+In addition to a fine under GDPR, the Irish government will also impose a fine up to 250,000 EUR per message sent by a company and an individual may be fined up to 50,000 EUR per message
```

Comment:
> Irish government will also impose a

I reckon this should be "can".

Reply:
https://github.com/threeheartsdigital/email-marketing-regulations/pull/29/commits/78a98b318bac8854628b7b8926245bd8dea6a535